packages/cli/src/commands/hydrogen/init.ts

```diff
@@ -6,27 +6,38 @@ import {
 import {renderFatalError} from '@shopify/cli-kit/node/ui';
 import Flags from '@oclif/core/lib/flags.js';
 import {output, path} from '@shopify/cli-kit';
-import {commonFlags} from '../../utils/flags.js';
+import {commonFlags, parseProcessFlags} from '../../utils/flags.js';
 import {transpileProject} from '../../utils/transpile-ts.js';
 import {getLatestTemplates} from '../../utils/template-downloader.js';
 import {readdir} from 'fs/promises';
 
+const STARTER_TEMPLATES = ['hello-world', 'demo-store'];
+const FLAG_MAP = {f: 'force'} as Record<string, string>;
+
 export default class Init extends Command {
   static description = 'Creates a new Hydrogen storefront.';
   static flags = {
-    path: commonFlags.path,
     force: commonFlags.force,
+    path: Flags.string({
+      description: 'The path to the directory of the new Hydrogen storefront.',
+      env: 'SHOPIFY_HYDROGEN_FLAG_PATH',
+    }),
     language: Flags.string({
       description: 'Sets the template language to use. One of `js` or `ts`.',
       choices: ['js', 'ts'],
-      default: 'js',
       env: 'SHOPIFY_HYDROGEN_FLAG_LANGUAGE',
     }),
     template: Flags.string({
       description:
         'Sets the template to use. One of `demo-store` or `hello-world`.',
+      choices: STARTER_TEMPLATES,
       env: 'SHOPIFY_HYDROGEN_FLAG_TEMPLATE',
     }),
+    'install-deps': Flags.boolean({
```

Comment:
Can you follow up with a PR to add these changes to the [shopify.dev docs.](https://shopify.dev/docs/custom-storefronts/hydrogen/cli/commands#init)